### DISCUSSION TITLE
Is this a good place to be holding discussions about things that are not yet fully-formed issues?

### DISCUSSION THREAD

**Discussion by geryan** [njtierney/pengweens#8] [General]
### Discussed in https://github.com/njtierney/pengweens/discussions/6

<div type='discussions-op-text'>

<sup>Originally posted by **geryan** February 13, 2024</sup>
E.g., before #3 is fully formed. 

Let's put a pin in it 🤘🏼</div>

**Comment by geryan** [comment]
ooh looky it's now an issue

**Comment by geryan** [comment]
It now has a new number - #7
I do not appear to be able to turn it back into a discussion however per #6

**Comment by njtierney** [comment]
Ah But I can!

**Comment by geryan** [comment]
And a new #8!
